### ISSUE TITLE
Bug: PASSWORD variable required for non-password VPN's

### ISSUE THREAD

**Issue by jnsn** [qdm12/private-internet-access-docker#175]
**TLDR**: Startup fails for Mullvad because a PASSWORD is required, but Mullvad doesn't have one

1. Is this urgent?

    - [x] Yes
    - [ ] No

2. What VPN service provider are you using?

    - [ ] PIA
    - [x] Mullvad
    - [ ] Windscribe
    - [ ] Surfshark
    - [ ] Cyberghost

3. What's the version of the program?

    **See the line at the top of your logs**

    `2020-06-18T19:49:59.937+0200    ERROR   no value found for environment variable "PASSWORD"`

4. What are you using to run the container?

    - [ ] Docker run
    - [x] Docker Compose
    - [ ] Kubernetes
    - [ ] Docker stack
    - [ ] Docker swarm
    - [ ] Podman
    - [ ] Other:

5. Extra information

Setting a dummy password value allows the container to start.

Logs:

```log
2020-06-18T19:49:59.908+0200    INFO    Unbound version: 1.9.6
2020-06-18T19:49:59.909+0200    INFO    IPtables version: v1.8.3
2020-06-18T19:49:59.932+0200    INFO    TinyProxy version: 1.10.0
2020-06-18T19:49:59.934+0200    INFO    ShadowSocks version: 3.3.4
2020-06-18T19:49:59.936+0200    INFO    OpenVPN version: 2.4.8
2020-06-18T19:49:59.937+0200    ERROR   no value found for environment variable "PASSWORD"
```

**Comment by qdm12** [comment]
Hello, thanks for taking the time to create an issue. It's fixed now, you don't need to specify a password for Mullvad. If you're in a hurry and don't want to wait for the build to finish (see actions above), you can alternatively put any password and it will work (FYI Mullvad uses a hardcoded `m` password for openvpn, which is forced by my program over any user given value). Happy tunneling!
